packages/fuels-programs/src/calls/call_handler.rs

```diff
@@ -198,7 +198,7 @@ where
                 .build(provider)
                 .await?;
 
-            provider.dry_run_opt(tx, false, None).await?
+            provider.dry_run_opt(tx, false, Some(0)).await?
```

Comment:
Can you clarify the difference between none and some(0)?

Reply:
We're telling the node to assume that the gas price is 0 for this dry run. `None` would mean it uses whatever the current price is.